### PR TITLE
Change ReadMe

### PR DESCRIPTION
**Pull request by Heleo2705**
[Api Client] Update README.md with usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,114 @@ All of the following exceptions defined in exceptions in this package have been
 4. `UnknownApiException` -> `UnknownApiError`
 5. `NoInternetException` -> `NoInternetError`
 6. `UnstableInternetException` -> `UnstableInternetError`
+
+## Introduction and Usage
+
+Fa_Flutter_API_Client is a customised implementation of API_client. It serves as a base for other FieldAssist apps.
+This package is used to implemented Dio effortlessly into other FieldAssist apps.
+
+The basic functionalities that this package offers are:-
```
Comment (apgapg) [inline README.md:32]: Reformat this para. It seems new line missing when reading. Please improve readability

File: README.md [hunk 2]
```diff
@@ -23,3 +23,114 @@ All of the following exceptions defined in exceptions in this package have been
 4. `UnknownApiException` -> `UnknownApiError`
 5. `NoInternetException` -> `NoInternetError`
 6. `UnstableInternetException` -> `UnstableInternetError`
+
+## Introduction and Usage
+
+Fa_Flutter_API_Client is a customised implementation of API_client. It serves as a base for other FieldAssist apps.
```
Comment (apgapg) [inline README.md:29]: must be in lowercase

**Review by apgapg — CHANGES_REQUESTED** [review]

**Comment by apgapg** [comment]
@Heleo2705 Please do the needful changes. Thanks in adv

File: README.md (hunk 1 above)
Comment (Heleo2705) [reply README.md:32]: Is it good now?

**Review by apgapg — APPROVED** [review]
LGTM

**Pull request merged**